### DISCUSSION TITLE
No Network for short Time after VPN is connected

### DISCUSSION THREAD

**Discussion by mewarnec** [schwabe/ics-openvpn#1463] [General]
After the VPN has connected, connections running over other interfaces are interrupted for short time? Anyone else have the same problem for e.g. that ADB disconnects after vpn has been connected.

**Comment by schwabe** [comment]
That is something from the OS. But I have only seen those disconnects for adb when using wireless adb. So prefer using adb via usb.

**Reply by mewarnec** [reply]
okay i am using adb over tcp over the usb/eth interface. There are some parralel connections to another device over ethernet which also interrupt for a second. Do you think there is a solution for preventing this disconnects without touching the os?

**Reply by schwabe** [reply]
To be honest, I never looked into it. USB adb always worked for me. Maybe try to exclude the network that you are using for adb from the VPN and see if that works.
